### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by jackpr-unity**
Adding a scene to test the legacy input system.  This one-panel scene sanity-checks kb, mouse, gamepad, button/axis mapping, UI, touchscreens, and IMU data.

### CODE REVIEW

File: Assets/QA/Tests/LegacyInputManager/Scripts/ImageRedIfMultiTouchEnabled.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Image))]
+public class ImageRedIfMultiTouchEnabled : MonoBehaviour {
```
Comment (Rene-Damm) [inline Assets/QA/Tests/LegacyInputManager/Scripts/ImageRedIfMultiTouchEnabled.cs]: Please run format.

**Review by Rene-Damm — CHANGES_REQUESTED** [review]

File: Assets/QA/Tests/LegacyInputManager/Scripts/ShowIMUData.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Text))]
+public class ShowIMUData : MonoBehaviour
```
Comment (Rene-Damm) [inline Assets/QA/Tests/LegacyInputManager/Scripts/ShowIMUData.cs]: What does IMU stand for?

File: ProjectSettings/EditorBuildSettings.asset [hunk 3]
```diff
@@ -6,6 +6,6 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/Demo/Demo.unity
-    guid: 3b868b211075646d5a6a9e07c61071b6
+    path: Assets/QA/Tests/LegacyInputManager/ISX_LegacyInputManager.unity
+    guid: 22a9cbef8d392f8409518c5174c25986
```
Comment (Rene-Damm) [inline ProjectSettings/EditorBuildSettings.asset]: Sorry didn't catch that before. Please revert this change.

File: Assets/QA/Tests/LegacyInputManager/Scripts/ShowIMUData.cs (hunk 2 above)
Comment (jackpr-unity) [reply Assets/QA/Tests/LegacyInputManager/Scripts/ShowIMUData.cs]: IMU = Inertial motion unit.  It's the term for the physical modules that measure and report acceleration, gyroscope, and magnetometer (compass) data.  Does Unity have a better term for grouping all this data together, or is this your way of telling me that my class name can't have non-standard acronyms?

**Review by Rene-Damm — CHANGES_REQUESTED** [review]

**Comment by jackpr-unity** [comment]
How does this fork look now - ok to pull in?

File: Assets/InputSystem.Extras/DualShock.meta [hunk 4]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: f14c62904b464a64a6dce94fc2ba0d9a
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (Rene-Damm) [inline Assets/InputSystem.Extras/DualShock.meta:8]: When committing with git, please stage only the files that you actually want to touch instead of staging every modified file.

File: ProjectSettings/DynamicsManager.asset [hunk 5]
```diff
@@ -14,16 +14,7 @@ PhysicsManager:
...
   m_LayerCollisionMatrix: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
   m_AutoSimulation: 1
   m_AutoSyncTransforms: 1
-  m_ClothInterCollisionSettingsToggle: 0
-  m_ContactPairsMode: 0
-  m_BroadphaseType: 0
-  m_WorldBounds:
-    m_Center: {x: 0, y: 0, z: 0}
-    m_Extent: {x: 250, y: 250, z: 250}
-  m_WorldSubdivisions: 8
```
Comment (Rene-Damm) [inline ProjectSettings/DynamicsManager.asset:29]: Same here as for comment above. This one downgrades ProjectSettings to a previous version of Unity.

File: Assets/InputSystem.Extras/DualShock.meta (hunk 4 above)
Comment (Rene-Damm) [reply Assets/InputSystem.Extras/DualShock.meta:8]: This one here adds a file back that was moved elsewhere.

**Review by Rene-Damm — APPROVED** [review]
I'll fix the unrelated changes after the merge.
